config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -604,6 +604,14 @@ def generate_misc():
                    test_timeout_minutes=150,
                    scenario="keypair-rotation",
                    extra_dashboards=['kops-misc']),
+        build_test(name_override="kops-aws-keypair-rotation-ha",
+                   cloud="aws",
+                   kops_channel="alpha",
+                   runs_per_day=1,
```

Comment:
In case we want to fix it, we need at least 3/day for now.
```suggestion
                   runs_per_day=3,
```